src/test/java/io/confluent/connect/elasticsearch/DataConverterTest.java

```diff
@@ -239,4 +245,63 @@ public void struct() {
     );
   }
 
+  @Test
+  public void ignoreNull() {
+    final String key = "key";
```

Comment:
Not following the static class suggestion. What would it contain and/or do?

Reply:
I was referring to just a POJO so you won't make all these local variables, private class members. But you can. Same thing, up to you.

Reply:
I'd recommend making these member variables initialized in a `@Before` method (e.g., `setup()`). Tests can use them without modification, but they can always just modify the ones that are needed. Of course, this would be different if all the tests used a different combination.